src/catalog/src/information_schema/views.rs

```diff
@@ -0,0 +1,288 @@
+// Copyright 2023 Greptime Team
```

Comment:
update `greptimedb/licenserc.toml` to 2024?

Reply:
No, just keep it, we'll update it in batch.